### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: './data/Au\\x_dom.npy'

### ISSUE THREAD

**Issue by y1216942674** [luo-ziyuan/Imaging-Interiors#2]
Hello, thank you very much for sharing your research findings. I would like to replicate your code, but I encountered an error related to the dataset files. Could you please provide the dataset and the processed files for the dataset?

**Comment by luo-ziyuan** [comment]
Hi, you can find the data downloading link in the README file, which is [this Google Drive link](https://drive.google.com/file/d/11pSTFg9QkP1Zjnl6Usc2dwJNMl4_8VcD/view?usp=sharing). Then, you can put the data under the ./data/ folder. Thank you! ^^

**Comment by y1216942674** [comment]
Hello, thank you very much for your response. I have a few questions after reading your code, and I hope you can help clarify them. The entire code uses only one scatterer in the training set, and the scatterer used for testing is also the same. In your paper, are all the settings based on training and testing with the same scatterer?

**Comment by luo-ziyuan** [comment]
Yes. We employ implicit neural representations (INR) to model individual scatterers, with each scatterer optimized separately. The training process relies solely on scattered field measurements, without utilizing groundtruth relative permittivity data.

**Comment by y1216942674** [comment]
Thank you very much for your reply. Your explanation has been very helpful to me. If it's convenient for you, could you please provide the code for your scatterer data simulation? We would like to build on your work for a new research project. If you agree, we will also publicly acknowledge you and your team.

**Comment by luo-ziyuan** [comment]
I'm currently preparing the code for release and plan to make it publicly available within 1-2 weeks. This part of code is developed based on [https://github.com/eleweiz/Solving-full-wave-nonlinear-inverse-scattering-problems-with-back-propagation-scheme](https://github.com/eleweiz/Solving-full-wave-nonlinear-inverse-scattering-problems-with-back-propagation-scheme). I will notify you when the code is public.

**Comment by y1216942674** [comment]
Hello, may I ask if the code for creating the dataset has been made public?

**Comment by luo-ziyuan** [comment]
Hi, you can find the data generation code in https://github.com/luo-ziyuan/Imaging-Interiors/tree/main/data_generation now.
